crates/sparrow-session/src/lib.rs

```diff
@@ -13,6 +13,7 @@ mod expr;
 pub mod partitioned;
 mod session;
 mod table;
+mod tests;
```

Comment:
I'd put `#[cfg(test)]` on the `mod tests`, then you don't need it on each individual test.